data/json/items/gun/9mm.json

```diff
@@ -995,8 +1022,17 @@
     "copy-from": "pistol_base",
     "looks_like": "glock_17",
     "type": "GUN",
-    "name": { "str": "M17" },
-    "description": "The M17 is a semi-automatic, short recoil operated pistol derived from the SIG Sauer P320.",
+    "name": { "str": "service handgun" },
+    "description": "The general issue military handgun of the United States army before the Cataclysm, this is a sizable pistol that fires 9x19mm cartridges.  Based off successful civilian designs and capable of taking magazines of various round capacities, the gun features a plastic polymer frame which helps to reduce overall weight.",
+    "variant_type": "gun",
+    "variants": [
+      {
+        "id": "m17",
+        "name": { "str": "M17" },
+        "description": "The M17 is a semi-automatic, short recoil operated pistol derived from the SIG Sauer P320.",
```

Comment:
I would have kept the military usage in this variant desc., like what you put for the M18.